files/open-dyslexic/info.ini

```diff
@@ -0,0 +1,4 @@
+author = "Abelardo Gonzalez"
+github = "https://github.com/mrkindy/jsdelivr/files/open-dyslexic"
```

Comment:
Usually this should point to a repository - this isn't good enough in this case. If theres a website we could substitute it for an empty string